cli/src/cli.rs

```diff
@@ -100,6 +100,10 @@ pub struct RunCmd {
 	#[structopt(long = "grandpa-pause", number_of_values(2))]
 	pub grandpa_pause: Vec<u32>,
 
+	/// Disable BEEFY
+	#[structopt(long = "disable-beefy")]
```

Comment:
```suggestion
	#[structopt(long = "no-beefy")]
```

Reply:
This became obsolete by just using `long`, I guess